contribs/emissions/src/main/java/org/matsim/contrib/emissions/ColdEmissionHandler.java

```diff
@@ -119,6 +119,7 @@ public void handleEvent(LinkLeaveEvent event) {
                     this.vehicleId2accumulatedDistance.put( vehicleId, distance );
                 }
                 // yyyy I have absolutely no clue what the distance stuff is doing here.  kai, jan'20
+                // I now think that this has to do with the fact that the cold emissions are smeared out over the initial distance.  Don't know the details, though.  kai, dec'22
```

Comment:
That is correct. We have two cold emission events. One right at the start of a trip and one after 1km.